cmd/ui/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"context"
+	"os/signal"
+	"syscall"
+
+	"github.com/abcxyz/jvs/pkg/ui"
+	"github.com/abcxyz/pkg/logging"
+)
+
+func main() {
```

Comment:
Just leaving a comment that Shan and I discussed replicating the logic from other projects, this is just a starter to get things working